docker.local/config/sc.yaml

```diff
@@ -48,11 +61,32 @@ smart_contracts:
     epoch: 15000000 # rounds
     # decline rewards every new epoch by this value (the block_reward)
     reward_decline_rate: 0.1 # [0; 1), 0.1 = 10%
+    # decline interests every new epoch by this value (the interest_rate)
+    interest_decline_rate: 0.1 # [0; 1), 0.1 = 10%
     # no mints after miner SC total mints reaches this boundary
     max_mint: 1500000.0 # tokens
     # if view change is false then reward round frequency is used to send rewards and interests
     reward_round_frequency: 250
-
+    cost:
+      add_miner: 100
+      add_sharder: 100
+      delete_miner: 100
+      delete_sharder: 100
+      miner_health_check: 100
+      sharder_health_check: 100
+      contributeMpk: 100
+      shareSignsOrShares: 100
```

Comment:
contributeMpk and shareSignsOrShares should be much higher than others, there are lots of signature verification there. We can check the smartcontracts benchmark result to set this appropriately.

Reply:
@peterlimg for sure, I think that I can use your latest performance results to find weights